addons/binding/org.openhab.binding.roku/src/main/java/org/openhab/binding/roku/RokuBindingConstants.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.roku;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link RokuBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Jarod Peters - Initial contribution
+ * @auther Shawn Wilsher - Overhaul of channels and properties
+ */
+public class RokuBindingConstants {
+
+    public static final String BINDING_ID = "roku";
+
+    // List of all Thing Type UIDs
+    public final static ThingTypeUID THING_TYPE_ROKU = new ThingTypeUID(BINDING_ID, "roku");
```

Comment:
Non-preferred order of modifiers (It was like this in a template, which has been changed some time ago) correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059